tests/models.py

```diff
@@ -62,13 +62,13 @@ class ChildModel(BaseModel):
 
 
 class InvalidFKRootModel(models.Model):
-    valid_fk = models.ForeignKey('InvalidFKNonRootModel', related_name='valid_non_root_fk', null=True, on_delete=models.CASCADE)
-    invalid_fk = models.ForeignKey('InvalidFKNonRootModel', related_name='invalid_non_root_fk', null=True, on_delete=models.CASCADE)
+    valid_fk = models.ForeignKey('InvalidFKNonRootModel', related_name='valid_non_root_fk', null=True, on_delete=models.CASCADE, db_constraint=False)
```

Comment:
Django 2.2 verifies foreign-keys on database-level, which means that the `test_invalid_foreign_key_doesnt_cause_matching_query_does_not_exist_exception` test fails without this change.